DOC_STRUCTURE.md

```diff
@@ -0,0 +1,88 @@
+# Documentation structure of the Babelfish Website
+
+## Documentation Index and folder structure
+
+The list of the documentation collection is configured in the `_config.yml` file, in the `just_the_docs` attribute. This is the original configuration:
```

Comment:
How about:

The website headings are configured in the `config.yml` file, in the `just_the_docs` attribute. Each value provided in a `name` attribute will be an expandable heading in the left navigation panel:

Reply:
Just a note: `_config.yaml` isn't a typo, keeping this.